datastore/longhorn.go

```diff
@@ -40,19 +37,10 @@ var (
 	VerificationRetryCounts   = 20
 )
 
-func (s *DataStore) InitSettings(settingPath string) error {
-	defaultSettingMap := map[string]string{}
-	if settingPath != "" {
-		data, err := ioutil.ReadFile(settingPath)
-		if err != nil {
-			return fmt.Errorf("failed to read default setting file %v: %v", settingPath, err)
-		}
-
-		if err := yaml.Unmarshal(data, &defaultSettingMap); err != nil {
-			logrus.Errorf("Failed to unmarshal customized default settings from yaml data %v, will use built-in internal default settings: %v", string(data), err)
-			// `yaml.Unmarshal()` can return a partial result. We shouldn't allow it
-			defaultSettingMap = map[string]string{}
-		}
+func (s *DataStore) InitSettings() error {
+	defaultSettings, err := types.GetCustomizedDefaultSettings()
```

Comment:
How about just `GetDefaultSettings` and fill the blank with the built-in default values?

Also, we don't want to read the file every time when we want to know what's the default setting is. It's better if we can load it when the daemon started.

Reply:
Do you mean using customized value to overwrite the built-in value in `InitSettings`? 

If yes, Can we add a bool variable to indicate that all default values are overwritten by the customized values? Then some daemon pods don't need to read the file.